q-transform/deltam.py

```diff
@@ -0,0 +1,6 @@
+def deltam():
+    """Fractional mismatch between neighbouring tiles
```

Comment:
Ok, some basic things. please follow the numpydoc style of documentation. See the pycbc/waveform/utils modules for some examples within this codebase. Also, please place functions into filter/qtransform.py